src/main/com/example009/radek/v2/Calculator.java

```diff
@@ -0,0 +1,51 @@
+package main.com.example009.radek.v2;
+
+class Calculator {
+
+    double result;
```

Comment:
proc mas jednu proměnnou package private a jednu protected?

Reply:
To protected ani nevím kde se vzalo, možná jsem to napsal a nesmazal. Obě jsem dal na private a dá se na ně přistupovat jen pomocí Getteru a setteru, abychom byli více systémoví, pokud jsem dobře pochopil princip.

Reply:
k propertam bys mel vzdy pristupovat pres gette,r ale pokud se pouziva v childovi, tak klidne protected 
casem na tu problematiku narazis